package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "contentful-management",
-  "version": "3.7.6",
+  "version": "4.0.0",
```

Comment:
this should be `4.0.0-rc1` `4.0.0-rc2` etc to not interfere with the `@latest` versions